package.json

```diff
@@ -44,7 +44,7 @@
     "@trevoreyre/autocomplete-js": "2.2.0",
     "@types/angular": "1.7.2",
     "@types/angular-animate": "1.5.10",
-    "@types/angular-dynamic-locale": "0.1.35",
+    "@types/angular-dynamic-locale": "0.1.34",
```

Comment:
Why this regression ?

Reply:
Good question :-)
Because on version 0.1.35 there is just no type in the package it just say that the types are now in the base package. But in the latest version of angular-dynamic-locale some types we use are missing, but on master it's more complete, everything we need is available...

Reply:
Ok thanks !